Extension/src/LanguageServer/extension.ts

```diff
@@ -82,10 +81,13 @@ async function initVcpkgDatabase(): Promise<vcpkgDatabase> {
                     });
                     reader.on('close', () => {
                         resolve(database);
-                        return;
                     });
                 });
             });
+            zipfile.on('end', () => { // In case the header db was not found
+                resolve({});
```

Comment:
You only want to resolve (or reject) if the promise has not been previously resolved.